sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/VirtualMachineScaleSetVMInstanceView.cs

```diff
@@ -58,7 +58,22 @@ public VirtualMachineScaleSetVMInstanceView()
         /// placementGroupId.</param>
-        public VirtualMachineScaleSetVMInstanceView(int? platformUpdateDomain = default(int?), int? platformFaultDomain = default(int?), string rdpThumbPrint = default(string), VirtualMachineAgentInstanceView vmAgent = default(VirtualMachineAgentInstanceView), MaintenanceRedeployStatus maintenanceRedeployStatus = default(MaintenanceRedeployStatus), IList<DiskInstanceView> disks = default(IList<DiskInstanceView>), IList<VirtualMachineExtensionInstanceView> extensions = default(IList<VirtualMachineExtensionInstanceView>), VirtualMachineHealthStatus vmHealth = default(VirtualMachineHealthStatus), BootDiagnosticsInstanceView bootDiagnostics = default(BootDiagnosticsInstanceView), IList<InstanceViewStatus> statuses = default(IList<InstanceViewStatus>), string assignedHost = default(string), string placementGroupId = default(string))
+        /// <param name="computerName">Specifies the host OS name of the
+        /// virtual machine. &lt;br&gt;&lt;br&gt; This name cannot be updated
+        /// after the VM is created. &lt;br&gt;&lt;br&gt; **Max-length
+        /// (Windows):** 15 characters &lt;br&gt;&lt;br&gt; **Max-length
+        /// (Linux):** 64 characters. &lt;br&gt;&lt;br&gt; For naming
+        /// conventions and restrictions see [Azure infrastructure services
+        /// implementation
+        /// guidelines](https://docs.microsoft.com/azure/virtual-machines/virtual-machines-linux-infrastructure-subscription-accounts-guidelines?toc=%2fazure%2fvirtual-machines%2flinux%2ftoc.json#1-naming-conventions).</param>
+        /// <param name="osName">The Operating System running on the hybrid
+        /// machine.</param>
+        /// <param name="osVersion">The version of Operating System running on
+        /// the hybrid machine.</param>
+        /// <param name="hyperVGeneration">The hypervisor generation of the
+        /// Virtual Machine [V1, V2]. Possible values include: 'V1',
+        /// 'V2'</param>
+        public VirtualMachineScaleSetVMInstanceView(int? platformUpdateDomain = default(int?), int? platformFaultDomain = default(int?), string rdpThumbPrint = default(string), VirtualMachineAgentInstanceView vmAgent = default(VirtualMachineAgentInstanceView), MaintenanceRedeployStatus maintenanceRedeployStatus = default(MaintenanceRedeployStatus), IList<DiskInstanceView> disks = default(IList<DiskInstanceView>), IList<VirtualMachineExtensionInstanceView> extensions = default(IList<VirtualMachineExtensionInstanceView>), VirtualMachineHealthStatus vmHealth = default(VirtualMachineHealthStatus), BootDiagnosticsInstanceView bootDiagnostics = default(BootDiagnosticsInstanceView), IList<InstanceViewStatus> statuses = default(IList<InstanceViewStatus>), string assignedHost = default(string), string placementGroupId = default(string), string computerName = default(string), string osName = default(string), string osVersion = default(string), string hyperVGeneration = default(string))
```

Comment:
Is this change manually done since the Specs PR: https://github.com/Azure/azure-rest-api-specs/pull/22218 is not merged yet? Please wait for the Specs PR to be merged, then regenerate the SDK in the PR. @ArcturusZhang for awareness as the assignee of the Specs PR.

Reply:
it has been merged @fengzhou-msft